### ISSUE TITLE
Package Uninstall instructions

### ISSUE THREAD

**Issue by chris-rock** [inspec/inspec#1338]
We should add uninstall instructions to our website. Those are analog to https://docs.chef.io/uninstall.html#chef-dk

**Comment by burtlo** [comment]
This issue is ...
![_157-](https://cloud.githubusercontent.com/assets/244426/21151157/16dd4708-c127-11e6-9e94-a59c5b24dc74.jpg)

**Comment by burtlo** [comment]
This feels like a tenant of good software that should not be closed.

**Comment by clebio** [comment]
The install/uninstall workflow raises more questions than it answers.

1. Why does it say `Sorry, try again` despite (a) a valid password and (b) continuing
1. why does it ask for a password at all
1. why does a `brew` operation just download and run the DMG installer
1. why did a clean install install an outdated version

```
$ brew cask uninstall inspec
==> Uninstalling Cask inspec
==> Uninstalling packages:
com.getchef.pkg.inspec
Password:
Password:
Sorry, try again.
==> Removing files:
/opt/inspec/
==> Purging files for version 4.6.3 of Cask inspec

$ brew cask install chef/chef/inspec
==> Satisfying dependencies
==> Downloading https://packages.chef.io/files/stable/inspec/4.6.3/mac_os_x/10.14/inspec-4.6.3-1.dmg
Already downloaded: /Users/caleb/Library/Caches/Homebrew/downloads/08d1c4cad0b5ca821fb75b9a7ff2585b391718ab74cbb0885a1949fb1181e555--inspec-4.6.3-1.dmg
==> Verifying SHA-256 checksum for Cask 'inspec'.
==> Installing Cask inspec
==> Running installer for inspec; your password may be necessary.
==> Package installers may write to any location; options such as --appdir are ignored.
installer: Package name is InSpec
installer: Installing at base path /
installer: The install was successful.
🍺  inspec was successfully installed!

$ inspec version
4.6.3

Your version of Chef InSpec is out of date! The latest version is 4.6.4.
```

**Comment by kekaichinose** [comment]
@clebio moved this over to #4442

**Comment by kekaichinose** [comment]
Closed in #4737
